src/World.cc

```diff
@@ -245,48 +250,51 @@ Errors World::Load(sdf::ElementPtr _sdf, const ParserConfig &_config)
 
   for (const auto &ifaceModelPair : this->dataPtr->interfaceModels)
   {
-    frameNames.insert(ifaceModelPair.second->Name());
+    implicitFrameNames.insert(ifaceModelPair.second->Name());
```

Comment:
add also to `nestedModelNames` (or `modelNames`)?

Reply:
Just to be sure, this would be different from the original code, which apparently didn't check if interface models had name collisions with regular models nor with other interface models.

Reply:
ok, I guess the frame graph should catch these name collisions. It wasn't obvious what needs to be checked by `implicitFrameNames`, so I suggested every inconsistency I could see

Reply:
Added a comment about what `implicitFrameNames` are used for.